bedrock-devnet/devnet/__init__.py

```diff
@@ -275,9 +275,11 @@ def devnet_deploy(paths):
     run_command(['docker', 'compose', 'up', '-d', 'da'], cwd=paths.ops_bedrock_dir, env=docker_env)
     wait_up(26658)
     time.sleep(15)
-    celestia_node_auth_token = run_command([
-        'docker', 'compose', 'exec', '--no-TTY', 'da', 'celestia', 'bridge', 'auth', 'admin', '--node.store', '/home/celestia/bridge'
-        ], cwd=paths.ops_bedrock_dir, env=docker_env, capture_output=True).stdout.decode().strip()
+    celestia_node_auth_token = os.getenv("CELESTIA_NODE_AUTH_TOKEN")
+    if not celestia_node_auth_token:
+        celestia_node_auth_token = run_command([
+            'docker', 'compose', 'exec', '--no-TTY', 'da', 'celestia', 'bridge', 'auth', 'admin', '--node.store', '/home/celestia/bridge'
```

Comment:
```suggestion
            'docker', 'compose', 'exec', '--no-TTY', 'da', 'celestia', 'light', 'auth', 'admin', '--node.store', '/home/celestia/light'
```

Reply:
although, not sure if this is ideal? i don't think it prevents anyone from using a light node and setting `CELESTIA_NODE_AUTH_TOKEN`

Reply:
Yep the user can replace local-celestia-devnet with their own light node, so they'd set manually `CELESTIA_NODE_AUTH_TOKEN` in that case.

Reply:
To clarify: the devnet script assumes that there's a local-celestia-devnet running, which has a bridge node. For the `Deploy an OP Stack devnet to Celestia` tutorial, we'll be modifying the same setup to post to celestia testnet instead. For this, the user will need to replace their local-celestia-devnet with celestia-node and this will require the user to manually configure the auth token as documented in `Set your Celestia node auth token` section here: https://github.com/celestiaorg/docs/pull/1566/files#diff-34f67718803f12b8e90305416f28ebb46cf94c6e41a5b7a1c811946d32a30654R207